server/apps/dictionaries/resource.py

```diff
@@ -38,8 +38,8 @@ class DictionariesResource(Resource):
         },
         'user': Resource.rel('users', nullable=True),
         'is_active': {
-            'type': 'boolean',
-            'default': True,
+            'type': 'string',
+            'default': 'true',
```

Comment:
This is related to the comment above. Wouldn't it be better to leave this as a boolean and perform the conversion to a string only when needed? (e.g. when sending a query to the backend)

It's more natural to work with booleans for properties that are, well, "naturally" boolean.